### ISSUE TITLE
package should not have angular as dependency (but devDependency)

### ISSUE THREAD

**Issue by Krisa** [akserg/ng2-dnd#50]
* **I'm submitting a ...**
[ x ] bug report
[ x ] feature request
[ ] question about the decisions made in the repository

* **Do you want to request a *feature* or report a *bug*?**
The version of the plugin is tied to a particular Angular version as we can see in the package.json (currently RC4). Therefore this plugin is using the RC4 version when resolving internal dependencies even so the main application may have a different version.
Current side effect, is that RC5 is a breaking update compared to RC4 and this plugin does not work in RC5 even so the base code is fully compatible - because as mentioned before, the plugin is using RC4 dependencies. In my case, I'm currently running my app in RC5 (or what is available so far from RC5) and I had to remove the node_modules in order to "fix" the issue (by removing node_modules, it resolves now automatically to my real own angular version).

Long story short, my suggestion would be to simply move the dependencies into devDependencies to avoid such issues.

**Comment by akserg** [comment]
Implemented in 1.12.0
